web/styles/zulip.css

```diff
@@ -2426,6 +2427,10 @@ select.invite-as {
     vertical-align: middle;
 }
 
+#deactivate-realm-user-modal .allowed_duration_message {
+    font-style: italic;
+}
```

Comment:
Would be better to have this CSS in `settings.css` since that is where we have the CSS for other modals opened from settings overlay.